src/styles/blocks/card.scss

```diff
@@ -0,0 +1,121 @@
+.card {
+  width: 198px;
+  height: 406px;
```

Comment:
not good way to fix height, remove it plz

Reply:
done. but when I duplicated my card, the test showed a problem with the button. but it works.